save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/projectcollection/CollectionView.kt

```diff
@@ -24,6 +27,25 @@ import react.dom.html.ReactHTML.td
 import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
 
+/**
+ * Enum that contains values for project
+ */
+enum class ProjectTab {
```

Comment:
To my mind it should have a different name like `ProjectsTab` or `ProjectWithPermissionsTab` in order to:
1. Tell `ProjectMenu` from `ProjectTab`
2. Highlight that this enum class is connected with visibility of projects

Reply:
Renamed